### PR TITLE
Added ability to specifying an eval statement for tabular data input. Useful for use cases such as combining multiple RFs to create a stress for comparison

### PR DESCRIPTION
**Pull request by stewartboyd119**
The eval statement allows a user to specify some combination
of ABAQUS historical data. The user specifies, in a normal fashion,
dictionaries definining how to interrogate the abaqus data with
a dictionary in the identifier list. The difference being that
an extra key value pair is provided *av_id* . This is a unique
identifier of the identifying object so that it may be referenced in
either the x_eval_statement or y_eval_statement (additional key, vals
in an element of results list).

x and y eval statements are strings which define a valid
python syntax equation for somehow combining the xy hist
data returned by the identifier. Internally a dictionary
d is defined mapping labels -> xydata.

Therefore an eval statement would be written as follows:

d['label1'] + d['label2']

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -162,7 +163,58 @@ A variety of different types of results can be extracted from the odbs and compa
 - `slope`: finds the slope of an xy data set
 - `finalValue`: finds the y value at the last increment in the xy data set
 - `x_at_peak_in_xy`: finds the x-value corresponding to the absolute peak in the y-value
-- `tabular`: Compares the values for a list of tuples specifying x, y points [(x1, y1), (x2, y2)...]
+- `tabular`: compares the values for a list of tuples specifying x, y points [(x1, y1), (x2, y2)...]. See example for further details
+
+### Tabular Example
+
+The tabular example by default uses the two identifier dict objects to define x and y data respectively (which is thusly compared to a
+list of tuples (specified as referenceValue). Additionally, a more advanced usage is allowed within the tabular option to specify a pythonic statement for combinging
```
Comment (acbergan) [inline readme.md]: I think a ')' is missing here.

File: readme.md [hunk 2]
```diff
@@ -162,7 +163,58 @@ A variety of different types of results can be extracted from the odbs and compa
...
+                    "position": "Node 4",
+                    "nset": "LOADFOLLOWERS"
+                },
+                {   "label": "y",
+                    "symbol": "RF2",
+                    "nset": "LOADAPP"
+                }
+            ],
+            # Use eval statements to calculate a reference strain and stress val from abaqus output of force and disp
+            "xEvalStatement": "(d['x1'] + d['x2']) / (2 * {l})".format(l=l),
```
Comment (acbergan) [inline readme.md]: Please rename 'l' since it looks almost identical to 1.

File: abaverify/processresults.py [hunk 3]
```diff
@@ -274,6 +274,7 @@ def write_results(results_to_write, fileName, depth=0):
     :type depth: int
     """
 
+    debug(fileName)
```
Comment (acbergan) [inline abaverify/processresults.py]: Is this needed?

**Review by acbergan — CHANGES_REQUESTED** [review]
Looks great in general! How do you feel about making the 'eval' stuff general so that it can be used with other results types? If you don't have the bandwidth to take this change on now, I'll accept the PR as is. Just a couple of nit-picks.

**Comment by stewartboyd119** [comment]
Generalized eval for other result types. Added eval test for max/min. Didn't do it initially for other result types as I didn't see a non eval equivalent in the test suite

**Comment by acbergan** [comment]
Looks great!

**Pull request merged**